x-pack/test/saved_object_api_integration/security_and_spaces/apis/find.ts

```diff
@@ -39,26 +47,30 @@ export default function ({ getService }: FtrProviderContext) {
   const createTests = (spaceId: string, user: TestUser) => {
     const currentSpaceCases = createTestCases(spaceId, []);
 
-    const explicitCrossSpace = createTestCases(spaceId, ['default', 'space_1', 'space_2']);
+    const EACH_SPACE = [DEFAULT_SPACE_ID, SPACE_1_ID, SPACE_2_ID];
+    const explicitCrossSpace = createTestCases(spaceId, EACH_SPACE);
     const wildcardCrossSpace = createTestCases(spaceId, ['*']);
 
-    if (user.username === 'elastic') {
+    if (user.username === AUTHENTICATION.SUPERUSER.username) {
       return {
         currentSpace: createTestDefinitions(currentSpaceCases.allTypes, false, { user }),
-        crossSpace: createTestDefinitions(explicitCrossSpace.allTypes, false, { user }),
+        crossSpace: [
+          createTestDefinitions(explicitCrossSpace.allTypes, false, { user }),
+          createTestDefinitions(wildcardCrossSpace.allTypes, false, { user }),
```

Comment:
👏

Reply:
Oh yeah, forgot to comment that I saw this was missing and added it. Good eye 😁